launchpod/angular-launchpod/backend/src/main/java/com/google/launchpod/data/UserFeed.java

```diff
@@ -1,16 +1,37 @@
 package com.google.launchpod.data;
 
 import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
 import com.google.launchpod.servlets.FormHandlerServlet;
 
 public final class UserFeed {
 
   private String xmlString;
+  private String title;
+  private String name;
+  private String rssLink;
+  private String description;
+  private String email;
+  private String postTime;
+  private long timestamp;
+  private Key key;
 
   private UserFeed(String xmlString) {
     this.xmlString = xmlString;
   }
 
+  public UserFeed(String title, String name, String rssLink, String description, String email, String postTime, long timestamp, Key key) {
```

Comment:
do we need both postTime and timeStamp? we can create post time from timeStamp when it's needed

Reply:
I don't believe the long datatype exists in Angular, so if that is the case we would need to send the postTime from the backend. The timestamp, however, is not needed here since the UserFeeds we are adding to the list are already being added in descending order. I have removed the timestamp.

Reply:
I have also removed the postTime from the datastore entity and it will be created when creating the list of userFeeds to send to the client.